src/pages/landing/ebooks/crypto.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Signup, WhatOurClientsSay } from '../../home/_lazy-load'
+import ImageText from './components/_image-text'
+import Topics from './components/_topics'
+import Hero from './components/_hero'
+import { Appearances } from 'components/custom/signup'
+import Layout from 'components/layout/layout'
+import { SEO } from 'components/containers'
+import { localize, WithIntl } from 'components/localization'
+import StocksIntroImage from 'images/svg/landing/crypto-intro.svg'
+
+const topicsCovered = [
+    localize('Introduction to cryptocurrencies'),
+    localize('Types of cryptocurrencies'),
+    localize('The basics of trading cryptocurrency'),
+    localize('Order types for derivatives trading'),
+    localize('How to evaluate a cryptocurrency'),
+    localize('Common crypto terms to impress your friends'),
+]
+
+const query = graphql`
+    query {
+        crypto_hero: file(relativePath: { eq: "ebooks/crypto-hero.png" }) {
+            ...bannerImage
+        }
+        crypto_inside: file(relativePath: { eq: "ebooks/crypto-inside.png" }) {
+            ...bannerImage
+        }
+    }
+`
+
+const StocksEbook = () => {
```

Comment:
Let's try to make any new files create a `.tsx` if there are JSX involved here. For the most basic files, we'll need to explicit declare our return type for our react function as such here. You can refer to some basic conventions we are following with moving to TS [here](https://wikijs.deriv.cloud/en/Frontend/deriv-static/06-implementation-guide/ts-convention).

```suggestion
const StocksEbook = (): ReactElement => {
```

Reply:
Hi @sean-binary , as discussed with dev and PM, this LP is urgent ,let's skip typescript for this.